tests/specs/api.e2e.js

```diff
@@ -99,4 +102,205 @@ describe('newrelic api', () => {
       expect(params.message).toEqual('too many free taco coupons')
     })
   })
+
+  describe('finished()', () => {
+    it('records a PageAction when called before RUM message', async () => {
+      const [insResult] = await Promise.all([
+        browser.testHandle.expectIns(),
+        browser.testHandle.expectRum(),
+        browser.url(await browser.testHandle.assetURL('api/finished.html', {
+          init: {
+            page_view_timing: {
+              enabled: false
+            },
+            ins: {
+              harvestTimeSeconds: 2
+            }
+          }
```

Comment:
I would check any config objects like this...if they are necessary I mean. The default feature harvest time is 5 seconds and that seems pretty stable for everyone and fast enough.